lib/node_modules/@stdlib/math/base/assert/is-even/src/addon.c

```diff
@@ -17,9 +17,11 @@
 */
 
 #include "stdlib/math/base/assert/is_even.h"
+#include "stdlib/napi/argv.h"
+#include "stdlib/napi/argv_double.h"
+#include "stdlib/napi/create_int32.h"
+#include "stdlib/napi/export.h"
 #include <node_api.h>
```

Comment:
```suggestion
#include <node_api.h>
#include <stdint.h>
```

Correction, it should be used below.